lib/src/extensions/image.dart

```diff
@@ -0,0 +1,73 @@
+import 'dart:typed_data';
+import 'dart:ui';
+
+import '../flame.dart';
+import 'color.dart';
+
+export 'dart:ui' show Image;
+
+extension ImageExtension on Image {
```

Comment:
Since you created this extension, there is a method that I think that would be cool to be here. We could have a `fromSprite` method, which would take a `Sprite` instance and return an `Image` instance which represents the "cropped" image that the sprite maps. What do you think?

Motivation: This would help on a new feature that I am planning on creating on fire atlas

Reply:
But in my use case, I would like to get the image from a Sprite, yeah, I could use composition, but this seems weird to me to use a class that compose an image, when I want to do the opposite operation. I don't think it would be such a redundant API since both classes have places which they fit better.

We could also put this on the Sprite class maybe, having a `toImage` method there.

Reply:
how about getCroppedImage() ? we can use ImageComposition underneath to impl it but I think having a method is useful!

Reply:
I have discussed this with @erickzanardo  privately as well and we concluded a method on the `Sprite` class would be useful indeed, and I'll be following up on that after this PR is merged.